plugins/nf-wave/src/main/io/seqera/wave/plugin/WaveClient.groovy

```diff
@@ -230,7 +235,8 @@ class WaveClient {
     }
 
     protected URL defaultFusionUrl() {
-        final isArm = config.containerPlatform()?.tokenize('/')?.contains('arm64')
+        // MARCO MARCO TO BE FIXED - platform is process specific
+        final isArm = false //config.containerPlatform()?.tokenize('/')?.contains('arm64')
```

Comment:
I think this can be solved by passing the container platform as parameter to `defaultFusionUrl` and `resolveContainerConfig`. 

When the platform is not available, like in the context of `sendRequest(String image)`, you can yse the default container platform.

Reply:
so, just to confirm, in `WaveRunCmd.groovy` the `runContainer()` method does not need to know about Fusion?

Reply:
Not directly. Maybe the best thing is to add a optional `platform` parameter that default to [DEFAULT_DOCKER_PLATFORM](https://github.com/nextflow-io/nextflow/blob/main/plugins/nf-wave/src/main/io/seqera/wave/plugin/WaveClient.groovy#L65-L65)

Reply:
now updating the unit tests

Reply:
(but having dinner soon)

Reply:
<img src="https://media0.giphy.com/media/vdbrUjzrUEGly/giphy.gif"/>

Reply:
😂

Reply:
Duff always helps .. done!